inlong-manager/pom.xml

```diff
@@ -60,6 +60,7 @@
         <gson.version>2.8.6</gson.version>
         <json-simple.version>1.1.1</json-simple.version>
         <jackson.version>2.9.10.5</jackson.version>
```

Comment:
I don’t know which version is more suitable, the jackson.version 2.9.5 version seems to have loopholes too.

Reply:
#2099 say the minimum repair version is `2.9.10.5`.

Reply:
In this [issue-https://github.com/FasterXML/jackson-databind/issues/3004](https://github.com/FasterXML/jackson-databind/issues/3004), it is pointed out that the safe version number is 2.9.10.8, please upgrade to this version.

Reply:
In this [issue-https://github.com/FasterXML/jackson-databind/issues/3004](https://github.com/FasterXML/jackson-databind/issues/3004), it is pointed out that the safe version number is 2.9.10.8, please upgrade to this version.